docs/application/dotnet/guides/nui/text.md

```diff
@@ -89,156 +97,89 @@ If no font is specified, default styles are used, and a suitable font for displa
 
 Setting a font size programmatically is not ideal for applications that support multiple screen resolutions, and for platforms that support multiple logical font sizes. In addition, making systemwide changes to your font settings override the font sizes that have been programmatically set.
 
-A more flexible approach is to prepare various JSON stylesheets and request a different style for each platform. The [Tizen.NUI.NUIApplication](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.NUIApplication.html) class has constructors that take a stylesheet argument:
+A more flexible approach is to prepare various JSON stylesheets and request a different style for each platform. To change the font style for standard text controls, create the `theme.json` file in the resources directory and use the following JSON syntax:
 
 ```csharp
-class Example : NUIApplication
-
-Example example = new Example("example-path/example.json");
+{
+    "styles": {
+      "CustomLabel": {
+        "fontFamily": "FreeSerif",
+        "fontStyle": {
+          "width": "expanded",
+          "weight": "bold",
+          "slant": "italic"
+        },
+        "pointSize": 12,
+        "textColor": [ 1, 1, 1, 1 ]
+      }
+    }
+}
 ```
 
-To change the font style for standard text controls, use the following JSON syntax:
+[StyleManager](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.StyleManager.html) provide the `ApplyTheme` method, where path to stylesheet is passed. Since stylesheet is loaded, `CustomLabel` style can be used on created label by using the `SetStyleName` method of `TextLabel`:

```

Comment:
Please change to:

```suggestion
[StyleManager](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.StyleManager.html) provides the `ApplyTheme` method, where path to stylesheet is passed. Since stylesheet is loaded, `CustomLabel` style can be used on the created label by using the `SetStyleName` method of `TextLabel`:
```